### ISSUE TITLE
Exception in pikepdf findall 

### ISSUE THREAD

**Issue by rsmuc** [pikepdf/pikepdf#23]
I installed the latest version of ocrmypdf (8.0.1-1) and pikepdf (1.0.5-1) in Manjaro (and also tried to use the versions from pypi) and receive the following exception:

```
 ERROR - Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/ruffus/task.py", line 748, in run_pooled_job_without_exceptions
    register_cleanup, touch_files_only)
  File "/usr/lib/python3.7/site-packages/ruffus/task.py", line 566, in job_wrapper_io_files
    ret_val = user_defined_work_func(*params)
  File "/usr/lib/python3.7/site-packages/ocrmypdf/_pipeline.py", line 867, in metadata_fixup
    not_copied = set(meta_original.keys()) - set(meta.keys())
  File "/usr/lib/python3.7/_collections_abc.py", line 720, in __iter__
    yield from self._mapping
  File "/usr/lib/python3.7/site-packages/pikepdf/models/metadata.py", line 496, in __iter__
    for node, attrib, _val, _parents in self._get_elements():
  File "/usr/lib/python3.7/site-packages/pikepdf/models/metadata.py", line 462, in _get_elements
    for rdfdesc in rdf.findall('rdf:Description[@rdf:about=""]', self.NS):
AttributeError: 'NoneType' object has no attribute 'findall'

```

After a downgrade to version 7.3.1 of ocrmypdf (including pikepdf 0.3.7) everything looks good.

Used ocrmypdf command: 
ocrmypdf 1.pdf 2.pdf -l deu

**Comment by jbarlow83** [comment]
Any chance you can provide the input file? I can easily fix the problem, but I'd like to see why your file causes this issue.

If you can't share the whole PDF, could you extract the XMP metadata object?

```python
import pikepdf
p = pikepdf.open('1.pdf')
data = p.Root.Metadata.read_bytes()
```

**Comment by rsmuc** [comment]
Seems to happen with every file created with SimpleScan on Linux. Here is an example file:
https://codeberg.org/rsmuc/temp/src/branch/master/example.pdf

**Comment by impredicative** [comment]
This bug is affecting me too with the example file http://nn.cs.utexas.edu/downloads/papers/stanley.ec02.pdf

**Comment by jbarlow83** [comment]
I already have the fix for this in the master branch (commit d31ea8fe), but I have not released it because my test suite currently fails on macOS for some arcane reason (exception handling with subclasses across library boundaries in the C++ ABI) that I haven't been able to figure out yet. I'm a little reluctant to create a release that won't work everywhere.

If this is a showstopper for you feel free to pull. You should be able to cherrypick the aforementioned commit without needing to recompile the C++ side since it's a Python only fix.

**Comment by jbarlow83** [comment]
Fixed in v1.1.0
